include/config.js

```diff
@@ -629,6 +630,12 @@ var BASE_CONFIG = {
             chain: null
         },
 
+		
+		//allow pencilblue to listen on a TCP IP port  pb.config.siteRoot+pb.config.sitePort
+		tcpip:true,
+		//allow pencilblue to create an UNIX socket
+		unixsocket:true,
```

Comment:
We can't assume that this is only being run on unix. It needs to be something that's turned on.